source/scale/deploy-prometheus-grafana-for-performance-monitoring.rst

```diff
@@ -100,20 +100,24 @@ Install Grafana
 
 .. tip:: 
 
-  - For troubleshooting advice, check the `Grafana Troubleshooting page <https://grafana.com/docs/grafana/latest/troubleshooting/>`__. 
-  - For user guides and tutorials, check the `Grafana documentation to learn more <https://grafana.com/docs/grafana/latest/>`__.
+  - For troubleshooting advice, check the `Grafana Troubleshooting page <https://grafana.com/docs/grafana/latest/troubleshooting/>`_. 
+  - For user guides and tutorials, check the `Grafana documentation to learn more <https://grafana.com/docs/grafana/latest/>`_.
 
 Getting started
 ---------------
 
 To help you get started, you can download three sample dashboards shared in Grafana:
 
-- `Mattermost Performance Monitoring v2 <https://grafana.com/grafana/dashboards/15582>`__, which contains detailed charts for performance monitoring including application, cluster, job server, and system metrics.
-- `Mattermost Collapsed Reply Threads Metrics <https://grafana.com/grafana/dashboards/15581>`__, which contains detailed metrics on the queries involved in our Collapsed Reply Threads feature.
-- `Mattermost Performance KPI Metrics <https://grafana.com/grafana/dashboards/2539>`__, which contains key metrics for monitoring performance and system health.
-- `Mattermost Performance Monitoring (Bonus Metrics) <https://grafana.com/grafana/dashboards/2545>`__, which contains additional metrics such as emails sent or files uploaded, which may be important to monitor in some deployments.
+.. tip::
+
+   See `this Grafana guide <https://grafana.com/docs/grafana/v7.5/dashboards/export-import/>`_ to learn how to import Grafana dashboards either from the UI or from the HTTP API.
 
-See `this Grafana guide <https://grafana.com/docs/grafana/v7.5/dashboards/export-import/>`__ to learn how to import Grafana dashboards either from the UI or from the HTTP API.
+- `Mattermost Performance Monitoring v2 <https://grafana.com/grafana/dashboards/15582>`_, which contains detailed charts for performance monitoring including application, cluster, job server, and system metrics.
+- `Mattermost Notification Health metrics <https://grafana.com/grafana/dashboards/21305-mattermost-notification-health>`_, which can be used to track different types of notifications sent from Mattermost. Accessing Mattermost Notification Health requires the feature flag ``NotificationMetrics`` to be set to ``true``. System admins can :ref:`disable notification metrics data collection <configure/environment-configuration-settings:enable notification metrics>` through the System Console.
+- `Mattermost WebApp Performance Metrics <https://grafana.com/grafana/dashboards/XXXXX>`_, which contains detailed metrics for client-side performance, including x, y, and z. Accessing Mattermost WebApp Performance Metrics requires the feature flag ``ClientMetrics`` to be set to ``true``.
```

Comment:
The URL for the dashboard isn't available yet, but it'll be available after https://github.com/mattermost/mattermost-performance-assets/pull/15 is merged.

As for the other details needed:
```suggestion
- `Mattermost Web App Performance Metrics <https://grafana.com/grafana/dashboards/XXXXX>`_, which contains detailed metrics for client-side performance, including web vitals and Mattermost-specifc metrics. Accessing Mattermost Web App Performance Metrics requires the feature flag ``ClientMetrics`` to be set to ``true``.
```